File: app/search_github.py
import urllib.request
import json
url = 'https://api.github.com/search/repositories?q=Domains'
req = urllib.request.Request(url, headers={'User-Agent': 'Mozilla/5.0'})
with urllib.request.urlopen(req) as response:
    data = json.loads(response.read().decode())
    for item in data['items']:
        if 'Astrae' in item['full_name'] or 'astrae' in item['full_name']:
            print(item['full_name'])

File: app/search_github.js
const https = require('https');
https.get('https://api.github.com/search/repositories?q=Domains', {headers: {'User-Agent': 'Node'}}, (res) => {
  let data = '';
  res.on('data', chunk => data += chunk);
  res.on('end', () => {
    try {
      const items = JSON.parse(data).items;
      items.forEach(i => {
        if(i.full_name.toLowerCase().includes('astrae')) console.log(i.full_name);
      });
    } catch(e) {}
  });
});
